### PR TITLE
feat(smtp): allow unauthorized mail server certificates

### PR DESCRIPTION
**Pull request by thecrafterjt**
New Branch for #513 
Needs testing!

### CODE REVIEW

File: backend/prisma/seed/config.seed.ts [hunk 1]
```diff
@@ -116,6 +116,12 @@ const configVariables: ConfigVariables = {
       defaultValue: "false",
       secret: false,
     },
+    allowUnauthenticatedCertificates: {
```
Comment (stonith404) [inline backend/prisma/seed/config.seed.ts:119]: I would rename `allowUnauthenticatedCertificates` to `allowUnauthorizedCertificates`

**Review by stonith404 — COMMENTED** [review]
Great, thanks for your contribution :) Please see my comment but else it looks good to me.

**Comment by thecrafterjt** [comment]
Updated Branch for new Config Entry Name

File: backend/prisma/seed/config.seed.ts (hunk 1 above)
Comment (thecrafterjt) [reply backend/prisma/seed/config.seed.ts]: Yeah, my bad, I even wanted to call it that, but I didn't :D

**Pull request merged**